proto/src/determined/api/v1/experiment.proto

```diff
@@ -544,7 +544,7 @@ message ExpMetricNamesResponse {
   repeated string training_metrics = 2 [deprecated = true];
   // List of validation metric names.
   repeated string validation_metrics = 3 [deprecated = true];
-  // List of metric type-name pairs.
+  // List of metric group-name pairs.
   repeated determined.metric.v1.MetricName metric_names = 4;
```

Comment:
MetricName in metric.proto looks like it still needs updating, right?

Reply:
yeah I missed that one. What should the comment be? 

```
// MetricName packages metric name and group.
message MetricName {
  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
    json_schema: { required: [ "group", "name" ] }
  };
  // The group of the metric.
  string group = 1;
  // The name of the metric.
  string name = 2;
}
```

also I'm thinking of naming this `MetricIdentifier` instead of metricname to match now-existing https://github.com/determined-ai/determined/blob/c7d473ebf7cb29c22cc8ffc71d28cefaf9124053/master/pkg/model/metrics.go#L61-L65